examples/index.js

```diff
@@ -2,37 +2,46 @@ const { Client, Collection } = require("discord.js");
 const { readdirSync } = require("fs");
 const { Manager } = require("erela.js");
 
-const client = new Client();
+const client = new Client({
+  intents: 32703,
```

Comment:
Can you explain what's intent `32703` is with a comment

Reply:
```
GUILDS 
GUILD_MEMBERS
GUILD_BANS
GUILD_EMOJIS_AND_STICKERS 
GUILD_INTEGRATIONS 
GUILD_WEBHOOKS 
GUILD_VOICE_STATES
GUILD_PRESENCES 
GUILD_MESSAGES 
GUILD_MESSAGE_REACTIONS 
GUILD_MESSAGE_TYPING 
DIRECT_MESSAGES 
DIRECT_MESSAGE_REACTIONS 
DIRECT_MESSAGE_TYPING 

```
 i am going to update the intents and remove the unused intents